packages/backend/test/event-images/post-event-image.test.ts

```diff
@@ -0,0 +1,380 @@
+          [nanoid(), eventId.rows[0].event_id, datetime.start, datetime.end]
+        )
+
+        for await (const sectionRow of indivEvent.sections) {
+          for await (const section of sectionRow) {
+            const sectionId = await client.query<{ event_section_id: string }>(
+              'insert into event_sections (event_section_id, event_datetime_id, event_section_row_position, event_section_column_position) values ($1, $2, $3, $4) returning event_section_id',
+              [nanoid(), datetimeId.rows[0].event_datetime_id, section.sectionRowPosition, section.sectionColumnPosition]
+            )
+
+            for await (const seatRow of section.seats) {
+              for await (const seat of seatRow) {
+                await client.query<event_seats>(
+                  'insert into event_seats (event_seat_id, event_section_id, event_seat_price, event_seat_row_position, event_seat_column_position) values ($1, $2, $3, $4, $5)',
+                  [nanoid(), sectionId.rows[0].event_section_id, seat.seatPrice, seat.seatRowPosition, seat.seatColumnPosition]
+                )
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+}
+
+void t.test('post event image', async t => {
+  const app = createServer()
+
+  t.teardown(async () => {
+    await app.close()
+    await client.end()
+  })
+
+  try {
+    await client.connect()
+    await mockEvent()
+  } catch (error) {
+    t.error(error)
+    t.fail()
+  }
+
+  void t.test('passing empty string as eventId', async t => {
+    try {
+      const response = await app.inject({
+        method: 'POST',
+        url: '/event-images/'
+      })
+
+      t.strictSame(response.statusCode, 400, 'status code from posting without eventId')
+      t.strictSame(response.json().message, 'params should have required property \'eventId\'', 'error message')
+    } catch (error) {
+      t.error(error)
+      t.fail()
+    }
+  })
+
+  void t.test('passing unknown eventId', async t => {
+    try {
+      const response = await app.inject({
+        method: 'POST',
+        url: '/event-images/unknown_eventId'
+      })
+
+      t.strictSame(response.statusCode, 404, 'status code from posting without unknown_eventId')
+      t.strictSame(response.json().message, 'event not found', 'error message')
+    } catch (error) {
+      t.error(error)
+      t.fail()
+    }
+  })
+
+  void t.test('posting image to existing event', async t => {
+    const form = new FormData()
+    form.append('image', createReadStream(resolve(__dirname, 'test-event-image.png')))
+
+    try {
+      const response = await app.inject({
+        method: 'POST',
+        url: '/event-images/posteventimagetest555',
+        payload: form,
+        headers: form.getHeaders()
+      })
+
+      t.strictSame(response.statusCode, 200, 'status code from posting')
+      t.strictSame(response.json().message, 'complete', 'message from posting')
```

Comment:
what are you posting?

Reply:
![image](https://user-images.githubusercontent.com/63894614/155135910-068d0dd3-c233-48ef-af56-ca4fd1b1c24a.png)